lib/elixir/lib/macro.ex

```diff
@@ -374,6 +369,26 @@ defmodule Macro do
       iex> Macro.escape({:unquote, [], [1]}, unquote: true)
       1
 
+  ## Options
+
+    * `:unquote` - when true, this function leaves `unquote/1` and
+      `unquote_splicing/1` statements unescaped, effectively unquoting
+      the contents on escape. This option is useful only when escaping
+      ASTs which may have quoted fragments in them. Defaults to false.
+
+    * `:prune_metadata` - when true, removes metadata from escaped AST
+      nodes. Note this option changes the semantics of escaped code and
+      it should only be used when escaping ASTs, never values. Defaults
+      to false.
+      
+      As an example, `ExUnit` stores the AST of every assertion, so when
+      an assertion fails we can show code snippets to users. Without this
+      option, each time the test module is compiled, we get a different
+      MD5 of the module byte code, because the AST contains metadata
```

Comment:
I think there is something missing in this part:
> because the AST contains metadata specific, such as counters, specific the compilation environment.

Reply:
One word too many! Tks, fixed!

Reply:
😬

But, might a **”to”** be missed too? Right now it reads “because the AST contains metadata specific the compilation environment”, should it be “because the AST contains metadata specific **to** the compilation environment”?

Reply:
Good catch again! :D